cli.py

```diff
@@ -20,13 +20,22 @@ def calculate(operation, integers):
   if (operation == 'average'):
     return calc.average(integers)
 
+
+  if (operation == 'cos'):
+    return calc.cos(integers)  
+
+  if (operation == 'm-cm'):
+  	return '\n'.join(map(lambda x:str(x), calc.m_to_cm(integers)))
+
   if (operation == 'tan'):
     return calc.tan(integers) 
  
 
  
   return 0
 
+
+  return 0
```

Comment:
the code is not merged correctly. This line is not needed. Also fix the merge conflicts once again as I merged your sine PR as well.

Reply:
Done